subxt/src/utils/mod.rs

```diff
@@ -76,3 +76,19 @@ unsafe impl<T> Sync for PhantomDataSendSync<T> {}
 /// with collections like BTreeMap. This has the same type params
 /// as `BTreeMap` which allows us to easily swap the two during codegen.
 pub type KeyedVec<K, V> = Vec<(K, V)>;
+
+/// This is a simplified version of Substrate's
+/// `frame_support::traits::Metadata::StorageVersion`.
+#[derive(
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    Debug,
+    scale_encode::EncodeAsType,
+    scale_decode::DecodeAsType,
+)]
+pub struct StorageVersion(u16);
```

Comment:
Since we return this to the user we should either:

1. Make the `u16` pub (that's probably what they care about but currently it's inaccessible, or
2. Just return a `u16` from `storage_version` (that's my preference; I don't think this struct adds anything)